src/main/ts/core/NpmTags.ts

```diff
@@ -22,7 +22,9 @@ export const pickTags = async (
   branchName: string, branchState: BranchState, version: Version, npmTags: () => Promise<NpmTags>
 ): Promise<[ string, ...string[] ]> => {
 
-  const branchTag = branchName.replace(/[^\w.]+/g, '-');
+  const branchTag = branchName
+    .replace('/npm_and_yarn/', '/') // Strip dependabot secondary prefix
```

Comment:
Should we make this more generic so it works with other stacks? If so, we can work backwards from the end. Here's an example of a python one:

`dependabot/pip/web/django-3.1.8`

Reply:
Beehive is very explicitly an NPM stack only thing atm (ie only works when you have a package.json), so going down that path felt like over-engineering to me.

Reply:
Yeah okay that's fair enough, I guess this just stood out to me as a bit fragile in general, but if I'm reading it correctly the worst case will just be an odd branch name

Reply:
Yes, that's the worse case that can happen in this case.